docs/guide/internationalization.md

```diff
@@ -1,60 +1,143 @@
 
-### Using multiple languages
+A version always has at least one translation which by default is the *initial/main* translation. Further versions can be added, but only for languages that have previously been [added to the global translation list](#adding-available-languages), that is a list of all languages available in the system. The maximum number of languages in the system is 64.
 
-The system allows for multiple language versions (translations) of a Content item to be created. Translations are created per version of the item, so each version of the content can have a different set of translations.
+Different translations of the same Content item can be edited separately. This means that different users can work on translations into different languages at the same time.
 
-At minimum, a version always has one translation which by default is the *initial/main* translation. Further versions can be added, but only for languages that have previously been   [added to the global translation list](https://doc.ez.no/display/USER/6.+Creating+content%2C+advanced#id-6.Creatingcontent,advanced-Addingtranslations), that is a list of all languages available in the system. The maximum number of languages in the system is 64.
+### Adding available languages
 
-Different translations of the same Content item can be edited separately. This means that different users can work on translations into different languages at the same time.
+The multilanguage system operates based on a global translation list that contains all languages available in the installation. Languages can be [added to this list from the Admin Panel](https://doc.ezplatform.com/projects/userguide/en/1.12/creating_content_advanced/#adding-translations) in the user interface. After adding a language be sure to dump all assets to the file system:
+
+```
+php app/console assetic:dump
+```
+
+**The new language must then be added to the [SiteAccess](siteaccess.md) configuration**. Once this is done, any user with proper permissions can create Content item versions in these languages in the user interface.
 
 ### Translatable and untranslatable Fields
 
-Language versions actually concern translating values of the Fields of a Content item. Every Field in its definition in a Content Type can be set to be Translatable or not.
+Language versions consist of translated values of the Content item's Fields. In the Content Type definition every Field is set to be Translatable or not.
+
+eZ Platform does not decide by itself which Fields can be translated and which cannot. For some Field values the need for a translation can be obvious, for example for the body of an article. In other cases, for instance images without text, numbers or e-mail addresses, translation is usually unnecessary. Despite that, eZ Platform gives you the possibility to mark any Field as translatable regardless of its Field Type. It is only your decision to exclude the translation possibility for those Fields where it makes no sense.
```

Comment:
Numbers could be translatable because the Decimal mark is not the same in every country. Or is it only in the rendering?
https://en.wikipedia.org/wiki/Decimal_mark